tests/playwright/sanity/device-mode.test.js

```diff
@@ -0,0 +1,22 @@
+const { test, expect } = require( '@playwright/test' );
+const WpAdminPage = require( '../pages/wp-admin-page' );
+
+test.describe( 'Device mode', () => {
+    test( 'Correct device mode is returned on Desktop', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo ),
+			editor = await wpAdmin.useElementorCleanPost(),
+			container = await editor.addElement( { elType: 'container' }, 'document' );
+
+		await editor.addWidget( 'heading', container );
+		await editor.publishAndViewPage();
+		await page.waitForSelector( '.e-con' );
+
+		const deviceMode = await page.evaluate( () => {
+			return elementorFrontend.getCurrentDeviceMode();
+		} );
+
+		const isInDesktopMode = 'desktop' === deviceMode;
+
```

Comment:
I believe it is good to test all device modes, especially for all the additional breakpoints. 

On frontend, see my code for the nav menu for reference: https://github.com/elementor/elementor-pro/blob/99e6e591e1a19e843152bd2181d2a3b900d0c095/tests/playwright/sanity/navMenu.test.js#L193. 

If you would like to test it in the editor, you can check the Nested Tabs tests.